resources/lib/playerinfo.py

```diff
@@ -178,10 +193,12 @@ def update_total(self):
         ''' Update the total video time '''
         try:
             self.total = self.getTotalTime()
+            if not self.total:
```

Comment:
We should keep the logic within a `try:`-block to a minimal.
So this ought to go into an `else:`-block after the `except:`-block.

Reply:
BTW If nothing is being played (i.e. it just finished or was stopped) we get a RuntimeError, so maybe this really ought to go in a `finally:`-block instead?

Reply:
I did this because `self.getTotalTime()` returns 0.0 in `onPlayBackStarted`

Reply:
I understand, but I think we need to do this:
```python
        try:
            self.total = self.getTotalTime()
        except RuntimeError:
            pass
        finally:
            if not self.total:
                self.total = 100
```

Reply:
I'll remove this.